cpp/include/cugraph_c/cugraph_api.h

```diff
@@ -127,7 +128,7 @@ cugraph_error_t cugraph_random_walks(const cugraph_raft_handle_t* ptr_handle,
                                      cugraph_rw_ret_t* ret);
 
 /* SG graph allocator*/
-cugraph_graph_envelope_t* cugraph_make_sg_graph(const cugraph_raft_handle_t* p_handle,
+cugraph_graph_envelope_t* cugraph_make_sg_graph(const cugraph_handle_t* p_handle,
```

Comment:
Is the idea that this will be replaced by `cugraph_sg_graph_create()`, or are they different?

Reply:
Yes.  I left the version that makes a graph envelope, it is currently used by the prototype.  My expectation is that we will migrate away from the envelope version since transpose will be more efficient overall.